### PR TITLE
Omnibus 25 Oct, 2022

### PR DESCRIPTION
**Pull request by arwer13**
Voting 25/10/2022.

1. Send DAI 732,710 from Lido DAO Treasury to the RCC multisig wallet `0xDE06d17Db9295Fa8c4082D4f73Ff81592A3aC437`
2. Send LDO 50,311 from Lido DAO Treasury to the RCC multisig wallet `0xDE06d17Db9295Fa8c4082D4f73Ff81592A3aC437`
3. Wrap 11 ETH from Lido DAO Treasury to WETH
4. Send 11 WETH from Lido DAO Treasury to the bloXroute address `0xea48ba2edefae9e4ddd43ea565aa8b9aa22baf08`

### CODE REVIEW

**Review by rkolpakov — APPROVED** [review]
lgtm

File: tests/test_2022_10_25.py [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+    validate_agent_execute_ether_wrap_event,
+)
+from utils.finance import ZERO_ADDRESS
+from brownie.network.transaction import TransactionReceipt
+
+
+dao_agent_address = "0x3e40D73EB977Dc6a537aF587D48316feE66E9C8c"
+lido_dao_token = "0x5A98FcBEA516Cf06857215779Fd812CA3beF1B32"
+weth_token = "0xC02aaA39b223FE8D0A0e5C4F27eAD9083C756Cc2"
+dai_token_address = "0x6B175474E89094C44Da98b954EedeAC495271d0F"
```
Comment (kadmil) [inline tests/test_2022_10_25.py:18]: Better either use dai fixture as `dai.address` or drop dai fixture & interface completely in favour for `interface.IERC20` provided by brownie

File: scripts/vote_2022_10_25.py [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+            reference="RCC Multisig Oct LDO Payout",
+        ),
+        # 3. Wrap 11 ETH from Lido DAO Treasury to WETH
+        encode_weth_wrap_agent_execute_call(agent, eth_amount),
+        # 4. Send 11 WETH from Lido DAO Treasury to the bloXroute address 0xea48ba2edefae9e4ddd43ea565aa8b9aa22baf08
+        make_weth_payout(
+            target_address=bloxroute_address,
+            weth_in_wei=eth_amount,
+            reference="Return 11 ETH back to bloXroute (in WETH form)",
+        ),
```
Comment (kadmil) [inline scripts/vote_2022_10_25.py:69]: Can we get us a util function to do just that maybe? It'll be used for most of eth transfers we're doing anyways (and it's at least the second time we're preparing such kind of eth transfer)

**Review by kadmil — COMMENTED** [review]
Left couple nitpicks here and there

File: tests/test_2022_10_25.py (hunk 1 above)
Comment (arwer13) [reply tests/test_2022_10_25.py:19]: fixed

File: scripts/vote_2022_10_25.py (hunk 2 above)
Comment (arwer13) [reply scripts/vote_2022_10_25.py:69]: it seems now there is no proper way to introduce an util function (discussed it)

**Pull request merged**